backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchHealthCheck.java

```diff
@@ -82,6 +82,7 @@ Result toHealthCheckResult(ClusterHealthResponse response) {
             case YELLOW:
                 return Result.healthy(COMPONENT_NAME);
```

Comment:
Please then log  at warn level for yellow

Reply:
Is this really necessary for ES? I remember we still return healthy here because yellow means that got only one instance and/or shard running (which we need to be valid for tests), thus we consider green and yellow the "same" here